spec/controllers/hub/organizations_controller_spec.rb

```diff
@@ -181,6 +181,7 @@
           name: "Oregano Organization",
           timezone: "America/Chicago",
           capacity_limit: "200",
+          allows_greeters: "true",
```

Comment:
This test sends `allows_greeters: "true"` in the POST, but since the factory will create the organization with `allows_greeters: "true"`, it seems to have no effect. Does it make sense to adjust the test to set `allows_greeters` to `false`?